tests/run-make/extern-flag-disambiguates/rmake.rs

```diff
@@ -0,0 +1,30 @@
+//@ ignore-cross-compile
+
+use run_make_support::{cwd, run, run_in_tmpdir, rustc};
+
+// Attempt to build this dependency tree:
+//
+//	A.1   A.2
+//	 |\    |
+//	 | \   |
+//        B  \  C
+//         \ | /
+//          \|/
+//           D
+//
+// Note that A.1 and A.2 are crates with the same name.
+
+fn main() {
+    run_in_tmpdir(|| {
```

Comment:
`run_in_tmpdir` is not actually needed, each test is executed in a fresh directory by default. This function is only needed when you need to create a temporary directory for a specific part of the test.